CHANGES.md

```diff
@@ -41,6 +41,11 @@
   [CMIP Dataset Ingestion Workflows](https://github.com/cedadev/stac-generator-example/tree/master/conf).
 
 - Adds ``optional-components/stac-public-access`` to give public access to the STAC catalog.
```

Comment:
Changes must be rebased on top with latest versions.

Reply:
I'll move those changes when the other PR is merged and after doing a new pull from master to get latest changes.